src/Options/Definitions.js

```diff
@@ -818,6 +818,13 @@ module.exports.PasswordPolicyOptions = {
       'Set the number of previous password that will not be allowed to be set as new password. If the option is not set or set to `0`, no previous passwords will be considered.<br><br>Valid values are >= `0` and <= `20`.<br>Default is `0`.',
     action: parsers.numberParser('maxPasswordHistory'),
   },
+  resetPasswordSuccessOnInvalidEmail: {
```

Comment:
I think we mostly call this "passwordReset" instead of "resetPassword", also the pages templates are named that way.

Could you rename this `passwordResetSuccessOnInvalidEmail`?